docs/src/solutions.md

```diff
@@ -249,6 +249,44 @@ lp_objective_perturbation_range
 lp_rhs_perturbation_range
 ```
 
+## Conflicts
+
+When the model you input is infeasible, some solvers can help you find the 
+cause of this infeasibility by offering a conflict, i.e., a subset of the 
+constraints that create this infeasibility.
```

Comment:
The standard name for this is irreducible inconsistent subsystem (IIS). Those key words should be used here in case someone searches the documentation for it.